webrtc.html

```diff
@@ -11591,7 +11591,76 @@ <h3>DTMF Example</h3>
   console.log('DTMF function not available');
 }
       </pre>
-    </section><!--
+    </section>
+    <section>
+      <h3>Perfect Negotiation Example</h3>
+      <p>Perfect negotiation is a recommended pattern to abstract away signaling
+      from the rest of an application. It lets applications operate on both
+      peer connection objects at once, on either end of the call, without having
+      to worry about signaling state, racing, or glare.</p>
```

Comment:
Can you add a definition (or just another sentence expanding upon) what "glare" means? This is a WebRTC-specific term that is never explained or mentioned in this document. You might also want to add a sentence clarifying what we may be concerned with to be "racing". You wouldn't need to worry about races if one side is always the offerer, for example. This helps clarifying what problems "perfect negotiation" solves.